apps/api/src/bff/api/endpoints/carts/handlers/get-cart.handler.ts

```
@@ -0,0 +1,16 @@
+import { IControllerHandler } from 'src/shared/controller-handler.interface';
+import { CartFacade } from 'src/modules/cart/application/cart.facade';
+import { CartReadModel } from 'src/modules/cart/application/query-handlers/read-models/cart.read-model';
+
+@Injectable()
+export class GetCartHandler implements IControllerHandler {
+  constructor(private readonly cartFacade: CartFacade) {}
+
+  async handle(
+    cartId: string,
+    userId: string,
+  ): Promise<CartReadModel> {
```

ESLint

[error] Replace `⏎····cartId:·string,⏎····userId:·string,⏎··` with `cartId:·string,·userId:·string`

(prettier/prettier)